docs/INSTALL-devel.rst

```diff
@@ -0,0 +1,290 @@
+
+
+If you wish to delete the user. do the following **as postgres user**::
+
+  dropuser haas
+
+**Note**: Make sure that the database role you create corresponds to an existing system user.
+eg. There has to be a system user `haas` to access database named `haas` as database role named `haas`.
+
+
+Create database `haas_dev` owned by database role `haas_dev`::
+
+  sudo -i -u haas_dev
+  -bash-4.2$ createdb haas
+
+Confirm it created a database named `haas_dev` and it is owned by `haas_dev`::
+
+
+  psql -c '\l'
+                                  List of databases
+    Name    |  Owner   | Encoding |   Collate   |    Ctype    |   Access privileges
+  -----------+----------+----------+-------------+-------------+-----------------------
+   haas_dev  | haas_dev | UTF8     | en_US.UTF-8 | en_US.UTF-8 |
+   postgres  | postgres | UTF8     | en_US.UTF-8 | en_US.UTF-8 |
+   template0 | postgres | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/postgres          +
+             |          |          |             |             | postgres=CTc/postgres
+   template1 | postgres | UTF8     | en_US.UTF-8 | en_US.UTF-8 | =c/postgres          +
+             |          |          |             |             | postgres=CTc/postgres
+
+
+switch to user `haas_dev`.
+All subsequent installation steps assumes you are in the 
+home directory of `haas_dev` 
+
+
+Getting Started with HaaS Installation
+======================================
+First you will need to fork and clone the HaaS repo into your dev VM.::
+
+  git clone https://github.com/**username**/haas.git
+  cd haas
+
+
+The first time you start working in the repository, set up a clean test
+environment::
+
+  virtualenv .venv
+
+Enter the environment (do this every time you start working with HaaS dev environment)::
+
+  source .venv/bin/activate
+
+Then, proceed with installing the HaaS and its dependencies into the virtual
+environment::
+
+  sudo python setup.py install
+  pip install -e .
+
+
+For older systems:
+==================
+
+On systems with older versions of ``pip``, such as Debian Wheezy and Ubuntu
+12.04, this install will fail with the following error::
+
+  AttributeError: 'NoneType' object has no attribute 'skip_requirements_regex'
+
+Fix this by upgrading ``pip`` within the virtual environment::
+
+  pip install --upgrade pip
+
+Versions of python prior to 2.7 don't have importlib as part of their
+standard library, but it is possible to install it separately. If you're
+using python 2.6 (which is what is available on CentOS 6, for example),
+you may need to run::
+
+  pip install importlib
+
+
+Setting up the Database:
+========================
+By default dev environment uses SQLite as a database, so if you're using it you can skip this section.
+
+If you wish to use PostgreSQL instead, you may get an error ``psycopg2 package not found``.
+  
+When you do ``haas-admin db create`` in the next step, you may need to install 
+following package on your system 
```

Comment:
Is this referring to the problem mentioned above? If so, maybe combine the sentences and clarify a bit. If these aren't the same problem, then when would one need to install the following package?
Why not include it in the giant list of packages you mention at the beginning?